spec/models/saml_request_spec.rb

```diff
@@ -75,8 +75,10 @@
       end
 
       it 'adds an error' do
+        err = 'Could not find any certificates to use. Please add a ' +
+              'certificate to your application configuration or paste one below.'
```

Comment:
heredoc
```suggestion
        err = <<~EOS.squish
          Could not find any certificates to use. Please add a
          certificate to your application configuration or paste one below.
        EOS
```

Reply:
updated in https://github.com/18F/identity-dashboard/pull/776/commits/8726d8eefc76b44db9fdc5eb9c75e8ae25dba821